install_bootstrap

```diff
@@ -283,7 +283,7 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
         # AWS questions            #
         ############################
         aws_default_region='aws_default_region'
-        ask_question "What is your AWS region?" "$AWS_DEFAULT_REGION" "AWS region" $aws_default_region
+        ask_question "What AWS region does dcc-dashboard reside in?" "$AWS_DEFAULT_REGION" "AWS region" $aws_default_region
```

Comment:
This region is for used for the location of the secrets manager. It's good you  reworded this, but I think you should say `What AWS region does the AWS Secrets Manager reside in?`.

Reply:
done